Lib/test/test_multiprocessing_with_big_output.py

```diff
@@ -0,0 +1,25 @@
+import unittest
```

Comment:
All multiprocessing tests should go into `_test_multiprocessing.py`, so that they get executed under the different execution contexts ("fork", "forkserver", "spawn").